pkg/roachprod/promhelperclient/client.go

```diff
@@ -37,8 +36,8 @@ import (
 const (
 	resourceName           = "instance-configs"
 	resourceVersion        = "v1"
-	serviceAccountJson     = "PROM_HELPER_SERVICE_ACCOUNT_JSON"
-	serviceAccountAudience = "PROM_HELPER_SERVICE_ACCOUNT_AUDIENCE"
+	ServiceAccountJson     = "PROM_HELPER_SERVICE_ACCOUNT_JSON"
+	ServiceAccountAudience = "PROM_HELPER_SERVICE_ACCOUNT_AUDIENCE"
```

Comment:
Should these have more generic names? It's strange to me that we'll need to say that in order to use `roachprod grafana-annotation`, you'll need to set the `PROM_HELPER_SERVICE_ACCOUNT_JSON` env var.